dascore/io/xml_binary/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+Module for reading data stored in xml_binary format.
+
+This format is a directory which contains metadata as a single xml file
+as well as any number of binary files (raw numeric buffers) which contain
+information about their start time in the file name. An example directory
+might look like this:
+
+data_folder
+   metadata.xml
```

Comment:
Currently, we only consider XML files named "metadata" so we should note this in the documentation. In the future, we may relax this restriction and look for any XML file in each directory, raising an error if there is more than one.

Reply:
I think we can devise a more general directory-based binary structure once we have support for an inventory-like thing. Let's defer to that point to address this.